janus_server/src/config.rs

```diff
@@ -29,6 +29,10 @@ pub trait BinaryConfig: Debug + DeserializeOwned {
 pub struct DbConfig {
     /// URL at which to connect to the database.
     pub url: Url,
+    /// Timeout in seconds to apply when creating, waiting for, or recycling
+    /// connection pool objects. This value will be used to construct a
+    /// `deadpool_postgres::Timeouts` value.
+    pub connection_pool_timeouts_secs: u64,
```

Comment:
Let's make this optional, defaulting to some reasonable value (60s SGTM) -- we should have a config knob for this, but I suspect we aren't going to want to turn it very often/ever.

Reply:
Good point. Annoyingly we can't just implement `std::default::Default` on `DbConfig` (what would we use for `url`?) and the `#[serde(default)]` annotation doesn't let us provide a literal, just the name of a function to call, so this takes a bit more code than I'd like but it's tolerable.